decidim-assemblies/app/views/decidim/assemblies/admin/assembly_user_roles/index.html.erb

```diff
@@ -9,52 +9,76 @@
-            <td>
+            <td data-label="<%= t("models.assembly_user_role.fields.name", scope: "decidim.admin") %>">
               <%= role.user.name %><br>
             </td>
-            <td>
+            <td data-label="<%= t("models.assembly_user_role.fields.email", scope: "decidim.admin") %>">
               <%= role.user.email %><br>
             </td>
-            <td>
+            <td data-label="<%= t("models.user.fields.last_sign_in_at", scope: "decidim.admin") %>">
               <% if role.user.invitation_sent_at %>
                 <%= l role.user.invitation_sent_at, format: :short %>
               <% end %>
             </td>
-            <td>
+            <td data-label="<%= t("models.user.fields.invitation_accepted_at", scope: "decidim.admin") %>">
               <% if role.user.invitation_accepted_at %>
                 <%= l role.user.invitation_accepted_at, format: :short %>
               <% end %>
             </td>
-            <td>
+            <td data-label="<%= t("models.user.fields.role", scope: "decidim.admin") %>">
               <%= t("#{role.role}", scope: "decidim.admin.models.assembly_user_role.roles") %><br>
             </td>
-            <td class="table-list__actions">
-              <% if allowed_to?(:invite, :assembly_user_role, user_role: role) && role.user.invited_to_sign_up? %>
-                <%= icon_link_to "refresh-line", resend_invitation_assembly_user_role_path(current_assembly, role), t("actions.resend_invitation", scope: "decidim.admin"), class: "resend-invitation", method: :post %>
-              <% end %>
+            <td class="table-list__actions" data-label="<%= t("models.assembly_user_role.fields.actions", scope: "decidim.admin") %>">
+              <button type="button" data-component="dropdown" data-target="actions-assembly-user-role-<%= role.id %>" aria-label="<%= t("decidim.admin.actions.actions_label", resource: role.user.name) %>">
+                <%= icon "more-fill", class: "text-secondary" %>
+              </button>
 
-              <% if allowed_to? :update, :assembly_user_role, user_role: role %>
-                <%= icon_link_to "pencil-line", edit_assembly_user_role_path(current_assembly, role), t("actions.edit", scope: "decidim.admin"), class: "action-icon--edit" %>
-              <% end %>
+              <div class="inline-block relative">
+                <ul id="actions-assembly-user-role-<%= role.id %>" class="dropdown dropdown__action" aria-hidden="true">
+                  <% if allowed_to?(:invite, :assembly_user_role, user_role: role) && role.user.invited_to_sign_up? %>
+                    <li class="dropdown__item">
+                      <%= link_to resend_invitation_assembly_user_role_path(current_assembly, role), method: :post, class: "dropdown__button resend-invitation" do %>
+                        <%= icon "refresh-line" %> <%= t("actions.resend_invitation", scope: "decidim.admin") %>
+                      <% end %>
+                    </li>
```

Comment:
Moving the HR in the conditional statement, fixes the appearance of black line in  the menu
![image](https://github.com/user-attachments/assets/23333cfb-121e-4b3e-9dda-1ce212a8923e)


```suggestion
                    </li>
                    <hr>
```